scripts/gen_vimdoc.py

```diff
@@ -710,10 +715,17 @@ def gen_docs(config):
         doc_file = os.path.join(base_dir, 'runtime', 'doc',
                                 CONFIG[mode]['filename'])
 
+        mpack_file = os.path.join(base_dir, 'runtime', 'doc',
+                                  CONFIG[mode]['filename'].replace('.txt',
+                                                                   '.mpack'))
+
         delete_lines_below(doc_file, CONFIG[mode]['section_start_token'])
         with open(doc_file, 'ab') as fp:
             fp.write(docs.encode('utf8'))
 
+        with open(mpack_file, 'ab') as fp:
```

Comment:
changed to `'wb'` since we want to overwrite any existing file, not append to it.